keras_retinanet/models/mobilenet.py

```diff
@@ -0,0 +1,82 @@
+"""
+Copyright 2017-2018 lvaleriu (https://github.com/lvaleriu/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import keras
+from keras.applications.mobilenet import MobileNet, BASE_WEIGHT_PATH, get_file, relu6, DepthwiseConv2D
+
+from ..models import retinanet
+
+mobile_net_custom_objects = {
+    'relu6': relu6,
+    'DepthwiseConv2D': DepthwiseConv2D
+}
+
+custom_objects = retinanet.custom_objects.copy()
+custom_objects.update(mobile_net_custom_objects)
+
+
+def download_imagenet(backbone):
+    alpha = float(backbone.split('_')[1])
+    rows = int(backbone.split('_')[0].replace('mobilenet', ''))
+
+    # load weights
+    if keras.backend.image_data_format() == 'channels_first':
+        raise ValueError('Weights for "channels_last" format '
+                         'are not available.')
+    if alpha == 1.0:
+        alpha_text = '1_0'
+    elif alpha == 0.75:
+        alpha_text = '7_5'
+    elif alpha == 0.50:
+        alpha_text = '5_0'
+    else:
+        alpha_text = '2_5'
+
+    model_name = 'mobilenet_{}_{}_tf_no_top.h5'.format(alpha_text, rows)
+    weight_path = BASE_WEIGHT_PATH + model_name
```

Comment:
What's the difference between `weight_path` and `weights_path` ?

Reply:
No difference - just a copy/paste. I'm removing it.

Reply:
"weight_path" is in fact "weights_url". Changing this!